public/js/utils/sources-tree.js

```diff
@@ -102,7 +103,7 @@ function getURL(source: TmpSource): { path: string, group: string } {
 function addToTree(tree: any, source: TmpSource) {
   const url = getURL(source);
 
-  if (IGNORED_URLS.includes(url) ||
+  if (includes(IGNORED_URLS, url) ||
```

Comment:
Can we just use `indexOf` instead of pulling in lodash again? I'd like to minimize the lodash deps and over time see if we really need it.